app/routes/contact.js

```diff
@@ -1,4 +1,29 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  model() {
+    return this.store.createRecord('contact');
+  },
+
+  actions: {
+    sendMessage(newContact) {
+      const controller = this.controller;
+      const model = controller.get('model');
+
+      newContact.save().then(() => {
+        alert(`We have received your message: "${newContact.get('message').trim()}"; \
+               we will reach out to you via "${newContact.get('emailAddress')}"`
+        );
+        controller.set('responseMessage', `We got your message and we'll get in touch soon`);
+        model.set('emailAddress', '');
```

Comment:
@SentientBicycle and @jasonly once the message is sent, I need to clear the two fields.  the following two lines do the trick but I'm not sure is the right way to do it.